.github/workflows/build-docs.yml

```diff
@@ -88,29 +88,48 @@ jobs:
         continue-on-error: true
         env:
           VERSION: ${{ github.event.release.tag_name }}
+          JVM_PROJECTS: |
+            jans-auth-server
+            jans-orm
+            jans-config-api
+            jans-scim
+            jans-core
+            jans-notify
+            jans-fido2
+            jans-eleven
+            agama
         run: |
          echo "Custom work on generating docs can go here."
           
          # Run cn docs 
          sudo bash ./automation/docs/generated-cn-docs.sh
           
-         # Generate property docs and push to main
+         echo "Generating property docs and push to main"
          sudo bash ./automation/docs/generate-property-docs.sh 
          cd docs
          git add . || echo "Nothing to add"
          git commit -a -S -m "docs: automated property doc generation" || echo "Nothing to commit"
-         git push
          cd ..
           
-         # Replace release number markers with actual release number
+         echo "Replacing release number markers with actual release number"
          if [ ! -z "$VERSION" ]; then
            cd docs
            egrep -lRZ --exclude=CONTRIBUTING.md . | xargs -0 -l sed -i -e "s/replace-janssen-version/$VERSION/g"
            git add . || echo "Nothing to add"
            git commit -a -S -m "docs: replace release marker with release number" || echo "Nothing to commit"
-           git push
            cd ..
          fi
+          
+         echo "Generating javadocs"
+          . ./automation/docs/generate-javadocs.sh
```

Comment:
I have removed the condition of `VERSION` from this block of code as I think generating javadocs at each push is ok. Hope this is fine.

Reply:
its fine .